mnemosyne/core/hygiene.py

```
@@ -243,46 +243,99 @@ def _scan_table(
+        "table_counts": table_counts,
+        "with_secrets": sum(1 for c in candidates if c.secret_flags),
+    }
+
+
 def audit_noise(
     db_path: Path,
     limit: int = 200,
     tables: Optional[List[str]] = None,
     min_score: float = 0.3,
+    offset: int = 0,
+    scan_all: bool = False,
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)